packages/cli/__tests__/compile-stylex-folder-test.js

```diff
@@ -15,44 +15,56 @@ import { compileDirectory } from '../src/transform';
 import * as cacheModule from '../src/cache';
 import { getDefaultCachePath } from '../src/cache';
 
-import fs from 'fs';
+const fs = require('fs').promises;
```

Comment:
```suggestion
const fs = require('fs/promises')
```

Reply:
current type definitions are outdated so flow is erroring on this, will keep for now